ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ReadKey();
+            Console.ForegroundColor = ConsoleColor.Green;
+               Console.WriteLine("Редьярд Киплинг. Заповедь");
+            {
+                Console.ReadKey();
+                Console.ForegroundColor = ConsoleColor.Yellow;
+                {
+                    Console.WriteLine("Владей собой среди толпы смятенной, тебя клянущей за смятенье всех,");
+                    Console.ReadKey();
+                    Console.WriteLine("Верь сам в себя, наперекор вселенной, и маловерным отпусти их грех");
+                    Console.ReadKey();
+                    Console.WriteLine("Пусть час не пробил, жди, не уставая,");
+                    Console.ReadKey();
+                    Console.WriteLine("Пусть лгут лжецы, не снисходи до них;");
+                    Console.ReadKey();
+                    Console.WriteLine("Умей прощать и не кажись, прощая,");
+                    Console.ReadKey();
+                    Console.WriteLine("Великодушней и мудрей других.");
+                    {
+                        Console.Clear();
+                        Console.ForegroundColor = ConsoleColor.White;
+                        Console.WriteLine("Редьярд Киплинг. Заповедь");
+                        Console.WriteLine("Владей собой среди толпы смятенной, тебя клянущей за смятенье всех,");
+                        Console.WriteLine("Верь сам в себя, наперекор вселенной, и маловерным отпусти их грех");
+                        Console.WriteLine("Пусть час не пробил, жди, не уставая,");
+                        Console.WriteLine("Пусть лгут лжецы, не снисходи до них;");
+                        Console.WriteLine("Умей прощать и не кажись, прощая,");
+                        Console.WriteLine("Великодушней и мудрей других.");
+                    }
+                }
+            }
+            ConsoleKeyInfo ConsoleKeyInfo = Console.ReadKey();
```

Comment:
Зачем объявлять переменную, если она потом не используется?
```suggestion
            Console.ReadKey();
```